include/cute/config.hpp

```diff
@@ -45,12 +45,13 @@
 #endif // CUTE_HOST_DEVICE, CUTE_DEVICE
 
 #if defined(CUTLASS_ENABLE_SYCL)
-// the flag ENABLE_NVPTX should be set to 1 for SYCL Nvidia backend and CUDA backend. However, this flag will be set to 0 for SYCL backend on non-Nvidia devices
+// The flag CUTE_ENABLE_NVPTX should be set to 1 for SYCL Nvidia backend and CUDA backend.
+// However, this flag will be set to 0 for SYCL backend on non-Nvidia devices
 #if defined(__SYCL_DEVICE_ONLY__) && defined(__NVPTX__)
```

Comment:
In SYCL when `__SYCL_CUDA_ARCH__` is enabled, this automatically means that ` __SYCL_DEVICE_ONLY__ && __NVPTX__` are enabled. So you can safely removes these flags all together and only use the `__SYCL_CUDA_ARCH__` instead of `CUTE_ENABLE_NVPTX` and `CUTLASS_ENABLE__NVPTX`